plugins/util.js

```diff
@@ -236,6 +287,99 @@ function generateIndexSearch({ posts }) {
   return indexJson;
 }
 
+/**
+ * getSitemapData
+ */
+
+function generateSitemap({ posts = [], pages = [] }) {
+  const { homepage = '' } = config;
+
+  const sitemap = `<?xml version="1.0" encoding="UTF-8"?>
+    <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+      <url>
+        <loc>${homepage}</loc>
+        <lastmod>${new Date().toISOString()}</lastmod>
+      </url>
+        ${pages
+          .map((slug) => {
+            return `<url>
+                      <loc>${homepage}/${slug}</loc>
+                      <priority>0.3</priority>
```

Comment:
is it common practice to not include the `lastmod` on these as well?

Reply:
My first thought was that pages are not modified as frequently as posts. But now at a second glance I think is a good idea to mantain coherence in the document. Will modify. Also here, do you agree to the idea of pages having little less priority than posts and homepage? Or should I remove priority tag and leave all urls with the same?

Reply:
yeah i think lower priority makes sense. im not an expert though on this, but the logic sounds right